### PR TITLE
docs: include source code styling section in docs style guide

### LINKED ISSUE #23691: Mention hex capitalization in author style guide
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ X ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

Neither Angular Author's Style Guide
https://angular.io/guide/docs-style-guide
nor Google Dev Doc Style Guide
https://developers.google.com/style/
are clear wrto capitalization of hex codes for colors

## Expected behavior

update https://angular.io/guide/docs-style-guide

## What is the motivation / use case for changing the behavior?

More complete guidelines for authors

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

### PR DESCRIPTION
**Pull request by josephperrott**
Include a source code styling section in the style guide for docs,
indicating to follow the Angular style guide where possible and
defining hexadecimal style guidance.

Closes #23691

### CODE REVIEW

File: aio/content/guide/docs-style-guide.md [hunk 1]
```diff
@@ -727,6 +727,13 @@ Remember to exclude these files from StackBlitz by listing them in the `stackbli
 
 <code-example path="docs-style-guide/stackblitz.json" header="stackblitz.json"></code-example>
 
+### Source code styling
+
+Source code should follow [Angular's style guide](guide/styleguide) where possible.
+
+#### Hexadecimals
```
Comment (petebacondarwin) [inline aio/content/guide/docs-style-guide.md:734]: Did you mean for this to be sub heading of `Source code styling`? I can't tell.

**Comment by petebacondarwin** [comment]
Is this PR fixing a specific issue?

**Comment by josephperrott** [comment]
@petebacondarwin Updated description with closing note.

File: aio/content/guide/docs-style-guide.md (hunk 1 above)
Comment (josephperrott) [reply aio/content/guide/docs-style-guide.md:734]: I did, though I could definitely be wrong with it being a subheading.

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**